modules/nextflow/src/main/resources/nextflow/executor/command-trace.txt

```diff
@@ -66,6 +66,14 @@ nxf_stat() {
 
     [ $NXF_DEBUG = 1 ] && echo -e "++ stat SUM=${sum[*]}"
 
+    ## use memory metrics from cgroup if available
+    if [ -e /sys/fs/cgroup ]; then
```

Comment:
I think that this may always exists, even if you are not in a container.

Reply:
I think I will just insert `true` or `false` from the bash wrapper builder based on whether the task is containerized.